recipes/smart_open/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "smart_open" %}
+{% set version = "1.3.5" %}
+{% set sha256 = "379bec2c08c518a30fe58e781abcae8d4e58bc0024c950f13d55ebcc899e3aba" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - boto >=2.32
+    - bz2file
+    - requests
+
+  run:
+    - python
+    - boto >=2.32
+    - bz2file
+    - requests
+
+
+test:
+  imports:
+    - smart_open
+    - smart_open.tests
+
+about:
+  home: https://github.com/RaRe-Technologies/smart_open
+  license: MIT
+  license_file: LICENSE
+  summary: 'A Python library for efficient streaming of large files'
+
+  # The remaining entries in this section are optional, but recommended
+  description: |
+    simplejson is a is a Python 2 & Python 3 library 
+    for efficient streaming of very large files
+    from/to S3, HDFS, WebHDFS or local (compressed) files. 
+  dev_url: https://github.com/RaRe-Technologies/smart_open
+
+extra:
+  recipe-maintainers:
+    - piskvorky
+    - tmylk
```

Comment:
Are you ok with being maintainers @piskvorky and @tmylk ?

Also care to add yourself too @souravsingh ?